plugins/jobs/src/js/JobsMenuClient.js

```diff
@@ -0,0 +1,32 @@
+import "rxjs/add/operator/combineLatest";
+import "rxjs/add/operator/map";
+
+import { makeExecutableSchema } from "graphql-tools/dist/index";
+import { runJob } from "#SRC/js/events/MetronomeClient";
+
+export const typeDefs = `
```

Comment:
Now that Daniel has this schema:
https://github.com/dcos/dcos-ui/pull/3006

I suggest we make this queries against that schema, and move our resolvers and mutations to that file (I believe it is a good idea to rebase from that branch).

Also the naming of this file is a little of. The files with extension client are usually network to data layer adapters. The files with graphql resolvers and type definitions are supposed to be Models. And we are aiming to have an unified model for jobs.

Reply:
Moving that I also think we should test the resolvers for
- returning a stream
- the stream has the transformations I want (which in this case might mean just that the runJob is being called with the id of the args I passed.